dexcom/event.go

```diff
@@ -176,9 +194,12 @@ func (e *Event) Parse(parser structure.ObjectParser) {
 	e.Type = parser.String("eventType")
 	e.SubType = parser.String("eventSubType")
 	e.Unit = parser.String("unit")
-	e.Value = parser.Float64("value")
-	e.ID = parser.String("eventId")
+	e.Value = parser.String("value")
```

Comment:
value now a string - so we need to parse as float form some of the validation checks below

Reply:
I wonder if the docs are correct, since the example below the API fields shows a numerical value: https://developer.dexcom.com/docs/dexcomv3/operation/getEventsV3/

Reply:
Put another way, do we need to accommodate this field being either a string or a float?

Reply:
from the data I have seen in the v3 sandbox, from all generations of devices,  value is now def now a string
e.g. 

` "value": "30.00",
   "unit": "grams",`